server/start.go

```diff
@@ -115,7 +115,13 @@ which accepts a path for the resulting pprof file.
 
 			// amino is needed here for backwards compatibility of REST routes
 			err := startInProcess(serverCtx, clientCtx, appCreator)
-			return err
+			errCode, ok := err.(ErrorCode)
+			if !ok {
+				return err
+			}
+
+			serverCtx.Logger.Info(fmt.Sprintf("received quit signal: %d", errCode.Code))
```

Comment:
maybe even Debug? ref: https://github.com/cosmos/cosmos-sdk/issues/8132

This is what I see now:
```
// -- snip --
12:16PM INF Stopping BlockchainReactor service impl={"Logger":{},"Switch":{"Logger":{}}} module=blockchain
12:16PM INF Saving AddrBook to file book=/Users/amaury/.simapp/config/addrbook.json module=p2p size=0
12:16PM ERR Stopped accept routine, as transport is closed module=p2p numPeers=0
12:16PM INF Closing rpc listener listener={"Listener":{}}
12:16PM INF RPC HTTP server stopped err="accept tcp 127.0.0.1:26657: use of closed network connection" module=rpc-server
12:16PM ERR Error serving server err="accept tcp 127.0.0.1:26657: use of closed network connection"
12:16PM INF exiting...
12:16PM INF received quit signal: 130
```

I guess the exiting is enough?

Reply:
Yeah I agree. Even better if we could remove the `exiting...` instead. tidbits though.